src/core/schema.js

```
@@ -59,43 +60,59 @@ function processPropertyDefinition (propDefinition) {
   }
 
   propType = propertyTypes[typeName];
-
   if (!propType) {
-    warn('Unknown property type: ' + typeName);
+    error('Unknown property type: ' + typeName);
     return propDefinition;
```

ok changed to throw rather than log